hyrax.gemspec

```diff
@@ -50,7 +50,7 @@ SUMMARY
   spec.add_dependency 'font-awesome-rails', '~> 4.2'
   spec.add_dependency 'hydra-derivatives', '~> 3.3'
   spec.add_dependency 'hydra-editor', '>= 3.3', '< 6.0'
-  spec.add_dependency 'hydra-head', '>= 10.6.1'
+  spec.add_dependency 'hydra-head', '>= 10.6.1', '< 12'
```

Comment:
should we do this separately

Reply:
Yeah. Good thought. I'll split this into two PRs.